src/wavesurfer.js

```diff
@@ -1168,6 +1173,23 @@ export default class WaveSurfer extends util.Observer {
         this.drawBuffer();
     }
 
+    /**
+     * Hide channels from being drawn on the waveform if splitting channels.
+     * 
+     * For example, if we want to draw only the peaks for the right stereo channel:
+     *
+     * const wavesurfer = new WaveSurfer.create({...splitChannels: true});
+     * wavesurfer.load('stereo_audio.mp3');
+     * 
+     * wavesurfer.setFilteredChannel([0]); <-- hide left channel peaks.
+     *
+     * @param {array} channelIndices Channels to be filtered out from drawing.
```

Comment:
can you add an `@version 4.0.0` to this method?

Reply:
done and done.